min_compatible_versions.csv

```diff
@@ -0,0 +1,189 @@
+This file was generated by scripts/min_compatible_versions.py
+pkg_name,min_version
+Flask-Cache,~=0.13.1
+Jinja2,~=2.11.0
+SQLAlchemy,==2.0.22
+WebTest,0
+Werkzeug,<1.0
+ai21,0
+aiobotocore,~=1.4.2
+aiofiles,0
+aiohttp,~=3.7
+aiohttp_jinja2,~=1.5.0
+aiomysql,~=0.1.0
+aiopg,~=0.16.0
+aiosqlite,0
+algoliasearch,~=2.5
+anthropic,~=0.28
+anyio,>=3.4.0
+aredis,0
+asgiref,~=3.0
+astunparse,0
+async_generator,~=1.10
+asyncpg,~=0.22.0
+asynctest,==0.13.0
+attrs,>=20
+austin-python,~=1.0
+blinker,0
+boto3,0
+botocore,~=1.13
+bottle,>=0.12
+bytecode,0

```

Comment:
here is one, in our tests we have `"bytecode": latest` for an iast test, but in `pyproject.toml` we have:

```
    "bytecode>=0.15.0; python_version>='3.12'",
    "bytecode>=0.14.0; python_version~='3.11.0'",
    "bytecode>=0.13.0; python_version<'3.11'",
```